worker/src/main/java/gov/cms/ab2d/worker/processor/ContractProcessorImpl.java

```diff
@@ -232,7 +232,7 @@ private int processHandles(List<Future<EobSearchResult>> futureHandles, Progress
     }
 
     private int writeOutResource(List<IBaseResource> eobs, StreamHelper helper, Versions.FhirVersions version) {
-        var jsonParser = fhirContext.newJsonParser();
+        var jsonParser = Versions.getContextFromVersion(version).newJsonParser();
```

Comment:
Is there any way to create these JSON parsers once? Creating them thousands of times doesn't make sense. Why not just create a parser for each version in the Versions class and pick the parser based on FhirVersion? If the json parsers are thread safe then this shouldn't be an issue.

Reply:
Yes, it shouldn't be here.